test/config/utility.cc

```diff
@@ -1177,6 +1178,32 @@ bool ConfigHelper::setListenerAccessLog(const std::string& filename, absl::strin
   return true;
 }
 
+void ConfigHelper::initializeTlsKeyLog(
+    envoy::extensions::transport_sockets::tls::v3::CommonTlsContext& tls_context,
+    bool keylog_local_filter, bool keylog_remote_filter) {
+  if (!keylog_local_filter && !keylog_local_filter) {
+    return;
+  }
+  auto tls_keylog_path = tls_context.mutable_tls_keylog()->mutable_logfile_path();
+#ifdef _MSC_VER
+  *tls_keylog_path = std::string("%temp%\\keylog");
+#else
+  *tls_keylog_path = std::string("/tmp/keylog");
+#endif
+  if (keylog_local_filter) {
+    auto tls_keylog_local = tls_context.mutable_tls_keylog()->mutable_local_address_range();
+    auto new_element_local = tls_keylog_local->Add();
+    new_element_local->set_address_prefix("127.0.0.1");
```

Comment:
This will fail on ipv6. I think it will work if you add `127.0.0.1` and `::1`. You can get these values from `getLoopbackAddressString()`.

Reply:
will fix